tests/Google/Ads/GoogleAds/Util/V2/GoogleAdsErrorsTest.php

```diff
@@ -93,14 +92,23 @@ public function testFromFailureWithInvalidOperationIndex()
     public function testFromFailureWithMutateOperation()
     {
         $failureWithMutateOperation = $this->createGoogleAdsFailure("mutate_operations");
-        
```

Comment:
Ditto

Reply:
Same (see [here](https://github.com/googleads/google-ads-php/blob/master/tests/Google/Ads/GoogleAds/Util/V1/GoogleAdsErrorsTest.php#L95))

Reply:
OK. Thanks for explaining.